src/collectors/hadoop/test/testhadoop.py

```diff
@@ -19,7 +19,7 @@ class TestHadoopCollector(CollectorTestCase):
 
     def setUp(self):
         config = get_collector_config('HadoopCollector', {
-            'metrics':  [os.path.dirname(__file__) + '/fixtures/*metrics.log'],
+            'metrics_files':  [os.path.dirname(__file__) + '/fixtures/*metrics.log'],
```

Comment:
dont see the new fixtures files though for the new format. Did we forget to git add?

Reply:
They were being git ignored, removed the ignore.